data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/handler/MethodNotAllowedHandler.java

```diff
@@ -22,6 +22,7 @@
 import org.slf4j.LoggerFactory;
 
 import static dev.knative.eventing.kafka.broker.core.utils.Logging.keyValue;
+import static dev.knative.eventing.kafka.broker.receiver.impl.handler.ProbeRequestUtil.*;
```

Comment:
the file above has explict static imports - I like that better, tbh

Reply:
Hmm, nice catch, let me setup my IDE properly